addons/mail/static/src/models/dropdown_menu_extra_item_view.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+/**
+ * Models a relation between a DropdownMenuView and a
+ * dropdownItemView where suggestable is used as iterating field for extra
+ * suggestions.
+ */
+registerModel({
+    name: 'DropdownMenuExtraItemView',
```

Comment:
same remark, name should be composer suggestion specific

Reply:
The model name was too long and too specific, when it's just about a showing of list partitioned as a main and extra part.

In the current state of this PR, this is still coupled to modelling of `DropdownMenu`, but the intent for follow-up work is to make listing of main/extra as a content of a popover view.

At the very least, very much don't want to keep using long model name `ComposerSuggestionListViewExtraComposerSuggestionViewItem`, which is too specific and is a readability nightmare for the code right now.